docs/src/joint/api/dia/Element/prototype/getAbsolutePointFromRelative.html

```diff
@@ -0,0 +1,6 @@
+<pre class="docs-method-signature"><code>element.getAbsolutePointFromRelative(x, y)</code></pre>
+<pre class="docs-method-signature"><code>element.getAbsolutePointFromRelative(relativePoint)</code></pre>
+
+<p>Accept a point in the coordinate system of the element and return the point in the graph coordinate system, represented as a <code>g.Point</code>.</p>
+
+<p>The element coordinate system has its origin <code>[0,0]</code> at the <code>element.position()</code> rotated by the <code>element.angle()</code>. Both axis <code>x</code> and <code>y</code> are rotated by the same angle.</p>
```

Comment:
I would say 'Each axis' here. 'Both axes' would be correct too, but I think people are more familiar with 'axis' instead of the plural 'axes'. The same change can be made in `docs/src/joint/api/dia/Element/prototype/getRelativePointFromAbsolute.html`